src/taskgraph/test/conftest.py

```diff
@@ -161,18 +162,20 @@ def make_fake_strategies():
 
 @pytest.fixture
 def transform_config(parameters):
-    graph_config = fake_load_graph_config(os.path.join("taskcluster", "ci"))
+    graph_config = fake_load_graph_config(str(here / "data" / "taskcluster" / "ci"))
```

Comment:
Is this directory supposed to exist, or is it a bogus path?

Reply:
It's a bogus path (for now at least). But it's needed so `GraphConfig.vcs_root` works:
https://github.com/taskcluster/taskgraph/blob/main/src/taskgraph/config.py#L110

I'm not very happy with that property.. but another problem for another time.